### DISCUSSION TITLE
expose "invoice metadata" field for ln addresses in API

### DISCUSSION THREAD

**Discussion by mp21-org** [btcpayserver/btcpayserver#6067] [Ideas / Feature Requests]
When creating a lightning address, we have a metaData field in the UI:
![grafik](https://github.com/btcpayserver/btcpayserver/assets/1374810/469e998c-45c0-4be3-990e-3b756baa412f)

However this is not exposed in the greenfield API:
https://docs.btcpayserver.org/API/Greenfield/v1/#operation/StoreLightningAddresses_AddOrUpdateStoreLightningAddress

Would be great to have it.

**Comment by dennisreimann** [comment]
Adressed in #6084.

**Comment by mp21-org** [comment]
closing since it was merged. Looking forward to the next release. 🥇
